src/typings/slickgrid.d.ts

```diff
@@ -1584,7 +1584,6 @@ declare namespace Slick {
 			public setPagingOptions(args: PagingOptions): void;
 			public getPagingInfo(): PagingOptions;
 			public getItems(): T[];
-			public getFilteredItems(): T[]; // manually adding this type - it's present in the definition but not the typings file from DefinitelyTyped
```

Comment:
Definitely keep this IMO - it's not going to affect anything to keep it around and will be useful to have while you work on the full refactor